File: ProductsApp/Connection.cs
using MySql.Data.MySqlClient;
using System;
namespace ProductsApp
{
    public class DBConnection
    {
        private readonly string _connectionString;
        private readonly string _TableName;
        private readonly string _PrimaryKeyName;
        private readonly string _TimeStampName;
        public DBConnection(string connectionString, string TableName, string PrimaryKeyName, string TimeStampName)
        {
            _connectionString = connectionString;
            _TableName = TableName;
            _PrimaryKeyName = PrimaryKeyName;
            _TimeStampName = TimeStampName;
        }
        public int Count
        {
            get
            {
                int Count = -1;
                try
                {
                    using (MySqlConnection connection = new MySqlConnection(_connectionString))
                    {
                        MySqlCommand command = new MySqlCommand($"SELECT COUNT({_PrimaryKeyName}) FROM {_TableName} ORDER BY {_PrimaryKeyName};", connection);
                        connection.Open();
                        MySqlDataReader reader = command.ExecuteReader();
                        while (reader.Read())
                        {
                            Count = reader.GetInt32(0);
                        }
                        reader.Close();
                        connection.Close();
                    }
                    return Count;
                }
                catch (Exception e)
                {
                    Console.WriteLine("DBConnection Count ERROR =\n\n" + e);
                    return Count;
                }
            }
        }
        public int[] Keys
        {
            get
            {
                int[] Keys = new int[Count];
                int counter = 0;
                try
                {
                    using (MySqlConnection connection = new MySqlConnection(_connectionString))
                    {
                        MySqlCommand command = new MySqlCommand($"SELECT {_PrimaryKeyName} FROM {_TableName} ORDER BY {_PrimaryKeyName}", connection);
                        connection.Open();
                        MySqlDataReader reader = command.ExecuteReader();
                        while (reader.Read())
                        {
                            Keys[counter++] = reader.GetInt32(0);
                        }
                        reader.Close();
                        connection.Close();
                    }
                    return Keys;
                }
                catch (Exception e)
                {
                    Console.WriteLine("DBConnection Keys ERROR =\n\n" + e);
                    return Keys;
                }
            }
        }
        public bool InsertRow(int Key, string Columns = "#NoData-?", string Values = "#NoData-?")
        {
            string FColumns = "";
            string FValues = "";
            if (Columns != "#NoData-?" || Values != "#NoData-?")
            {
                FColumns = $", {Columns}";
                FValues = $", {Values}";
            }
            using (MySqlConnection connection = new MySqlConnection(_connectionString))
            {
                try
                {
                    MySqlCommand command = new MySqlCommand($"INSERT INTO {_TableName} ({_PrimaryKeyName}, {_TimeStampName} {FColumns}) VALUES ({Key}, CURRENT_TIMESTAMP {FValues});", connection);
                    connection.Open();
                    command.ExecuteNonQuery();
                    connection.Close();
                    return true;
                }
                catch (Exception e)
                {
                    Console.WriteLine("DBConnection InsertRow ERROR =\n\n" + e);
                    return false;
                }
            }
        }
        public bool DeleteRow(int Key)
        {
            using (MySqlConnection connection = new MySqlConnection(_connectionString))
            {
                try
                {
                    MySqlCommand command = new MySqlCommand($"DELETE FROM {_TableName} WHERE {_PrimaryKeyName} = {Key};", connection);
                    connection.Open();
                    command.ExecuteNonQuery();
                    connection.Close();
                    return true;
                }
                catch (Exception e)
                {
                    Console.WriteLine("DBConnection DeleteRow ERROR =\n\n" + e);
                    return false;
                }
            }
        }
        public bool SetField<T>(int Key, string Colunm, T Value, bool Time = false)
        {
            string FValue = "";
            if (Value is byte || Value is short || Value is int || Value is long || Value is decimal || Value is decimal || Value is decimal)
            {
                FValue = Value.ToString();
            }
            else
            {
                FValue = $"'{Value.ToString()}'";
            }
            string FTime = "";
            if (Time)
            {
                FTime = $", {_TimeStampName} = CURRENT_TIMESTAMP";
            }
            using (MySqlConnection connection = new MySqlConnection(_connectionString))
            {
                try
                {
                    MySqlCommand command = new MySqlCommand($"UPDATE {_TableName} SET {Colunm} = {FValue}{FTime} Where {_PrimaryKeyName} = {Key};", connection);
                    connection.Open();
                    command.ExecuteNonQuery();
                    connection.Close();
                    return true;
                }
                catch (Exception e)
                {
                    Console.WriteLine("DBConnection SetField ERROR =\n\n" + e);
                    return false;
                }
            }
        }
        public string GetString(int Key, string Colunm)
        {
            string Field = "#NoData-?";
            try
            {
                using (MySqlConnection connection = new MySqlConnection(_connectionString))
                {
                    MySqlCommand command = new MySqlCommand($"SELECT {Colunm} FROM {_TableName} WHERE {_PrimaryKeyName} = {Key};", connection);
                    connection.Open();
                    Field = (string)command.ExecuteScalar();
                    connection.Close();
                }
                return Field;
            }
            catch (Exception e)
            {
                Console.WriteLine("DBConnection GetString ERROR =\n\n" + e);
                return Field;
            }
        }
        public int GetInt(int Key, string Colunm)
        {
            int Field = 0;
            try
            {
                using (MySqlConnection connection = new MySqlConnection(_connectionString))
                {
                    MySqlCommand command = new MySqlCommand($"SELECT {Colunm} FROM {_TableName} WHERE {_PrimaryKeyName} = {Key};", connection);
                    connection.Open();
                    Field = (int)command.ExecuteScalar();
                    connection.Close();
                }
                return Field;
            }
            catch (Exception e)
            {
                Console.WriteLine("DBConnection GetInt ERROR =\n\n" + e);
                return Field;
            }
        }
        public decimal GetDecimal(int Key, string Colunm)
        {
            Decimal Field = 0;
            try
            {
                using (MySqlConnection connection = new MySqlConnection(_connectionString))
                {
                    MySqlCommand command = new MySqlCommand($"SELECT {Colunm} FROM {_TableName} WHERE {_PrimaryKeyName} = {Key};", connection);
                    connection.Open();
                    Field = (decimal)command.ExecuteScalar();
                    connection.Close();
                }
                return Field;
            }
            catch (Exception e)
            {
                Console.WriteLine("DBConnection GetDecimal ERROR =\n\n" + e);
                return Field;
            }
        }
        public DateTime GetDateTime(int Key, string Colunm)
        {
            DateTime Field = new DateTime();
            try
            {
                using (MySqlConnection connection = new MySqlConnection(_connectionString))
                {
                    MySqlCommand command = new MySqlCommand($"SELECT {Colunm} FROM {_TableName} WHERE {_PrimaryKeyName} = {Key};", connection);
                    connection.Open();
                    Field = (DateTime)command.ExecuteScalar();
                    connection.Close();
                }
                return Field;
            }
            catch (Exception e)
            {
                Console.WriteLine("DBConnection GetDateTime ERROR =\n\n" + e);
                return Field;
            }
        }
    }
}

File: StoreWeb/src/Connection.php
<?php
class DBConnection
{
    protected $_servername;
    protected $_username;
    protected $_password;
    protected $_dbname;
    protected $_TableName;
    protected $_PrimaryKeyName;
    protected $_TimeStampName;
    function __construct($servername, $username, $password, $dbname, $TableName, $PrimaryKeyName, $TimeStampName, $CreateTable = false, $TableStructureQuery = "")
    {
        $this->_servername = $servername;
        $this->_username = $username;
        $this->_password = $password;
        $this->_dbname = $dbname;
        $this->_TableName = $TableName;
        $this->_PrimaryKeyName = $PrimaryKeyName;
        $this->_TimeStampName = $TimeStampName;
        if ($CreateTable) {
            $conn = new mysqli($this->_servername, $this->_username, $this->_password, $this->_dbname);
            if ($conn->connect_error) {
                die($conn->connect_error);
            };
            $conn->query($TableStructureQuery);
            $conn->close();
        }
    }
    function Count()
    {
        $conn = new mysqli($this->_servername, $this->_username, $this->_password, $this->_dbname);
        if ($conn->connect_error) {
            die($conn->connect_error);
        };
        $result = $conn->query("SELECT COUNT($this->_PrimaryKeyName) FROM $this->_TableName;");
        $Count = $result->fetch_assoc();
        $conn->close();
        return (int)$Count["COUNT($this->_PrimaryKeyName)"];
    }
    function Keys()
    {
        $Keys = null;
        $conn = new mysqli($this->_servername, $this->_username, $this->_password, $this->_dbname);
        if ($conn->connect_error) {
            die($conn->connect_error);
        };
        $result = $conn->query("SELECT $this->_PrimaryKeyName FROM $this->_TableName ORDER BY $this->_PrimaryKeyName;");
        $counter = 0;
        while ($row = $result->fetch_assoc()) {
            $Keys[$counter++] = $row["$this->_PrimaryKeyName"];
        }
        $conn->close();
        return $Keys;
    }
    function InsertRow($Key, $Columns = "#NoData-?", $Values = "#NoData-?", $Time = "#NoData-?")
    {
        $FColumns = "";
        $FValores = "";
        if ($Columns != "#NoData-?" || $Values != "#NoData-?") {
            $FColumns = ", $Columns";
            $FValores = ", $Values";
        }
        $conn = new mysqli($this->_servername, $this->_username, $this->_password, $this->_dbname);
        if ($conn->connect_error) {
            die($conn->connect_error);
        };
        if ($Time == "#NoData-?") {
            $Time = "CURRENT_TIMESTAMP";
        }
        $Successfully = $conn->query("INSERT INTO $this->_TableName ($this->_PrimaryKeyName, $this->_TimeStampName $FColumns)
        VALUES ($Key, $Time $FValores);");
        $conn->close();
        return $Successfully;
    }
    function DeleteRow($Key)
    {
        $conn = new mysqli($this->_servername, $this->_username, $this->_password, $this->_dbname);
        if ($conn->connect_error) {
            die($conn->connect_error);
        };
        $Successfully = $conn->query("DELETE FROM $this->_TableName WHERE $this->_PrimaryKeyName = $Key;");
        $conn->close();
        return $Successfully;
    }
    function SetField($Key, $Column, $Value, $Time = false, $KeyName = "#NoData-?", $ComplexKey = false)
    {
        $FKey = $Key;
        $FKeyName = $KeyName;
        if (!is_numeric($Key) && !$ComplexKey) {
            $FKey = "'$Key'";
        }
        if ($KeyName == "#NoData-?") {
            $FKeyName = $this->_PrimaryKeyName;
        }
        $FValue = $Value;
        if (!is_numeric($Value)) {
            $FValue = "'$Value'";
        }
        $conn = new mysqli($this->_servername, $this->_username, $this->_password, $this->_dbname);
        if ($conn->connect_error) {
            die($conn->connect_error);
        };
        $FTime = "";
        if ($Time) {
            $FTime = ", $this->_TimeStampName = CURRENT_TIMESTAMP";
        }
        $Successfully = $conn->query("UPDATE $this->_TableName SET $Column = $FValue $FTime WHERE $FKeyName = $FKey;");
        $conn->close();
        return $Successfully;
    }
    function GetField($Key, $Column, $ReturnArray = false, $KeyName = "#NoData-?", $OrderBy =  "#NoData-?")
    {
        $FKey = $Key;
        $FKeyName = $KeyName;
        $FOrderBy = $OrderBy;
        if (!is_numeric($Key)) {
            $FKey = "'$Key'";
        }
        if ($KeyName == "#NoData-?") {
            $FKeyName = $this->_PrimaryKeyName;
        }
        $FColumn = "$Column";
        $Where = "WHERE $FKeyName = $FKey";
        if ($Key == "#Distinct") {
            $FColumn = "DISTINCT($Column)";
            $Where = "";
        }
        if ($OrderBy == "#NoData-?") {
            $FOrderBy = $Column;
        }
        $Fields = null;
        $conn = new mysqli($this->_servername, $this->_username, $this->_password, $this->_dbname);
        if ($conn->connect_error) {
            die($conn->connect_error);
        };
        $result = $conn->query("SELECT $FColumn FROM $this->_TableName $Where ORDER BY $FOrderBy;");
        $counter = 0;
        while ($row = $result->fetch_assoc()) {
            $Fields[$counter++] = $row["$Column"];
        }
        $conn->close();
        if ($ReturnArray) {
            return $Fields;
        } else {
            return $Fields[0];
        }
    }
}
